Assets/Scripts/Settings.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Audio;
+
+public class Settings : MonoBehaviour
+{
+    public AudioMixer am;
```

Comment:
Public fields start with uppercase.
If you only need reference use private serialized field.